modules/node-observability-create-custom-resource.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/understanding-node-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="creating-node-observability-custom-resource_{context}"]
+= Creating the Node Observability custom resource
+
+Before you run profiling queries, you must create a `NodeObservability` custom resource (CR).
+
+[IMPORTANT]
+====
+Creating a `NodeObservability` CR reboots all the worker nodes. It might take 10 or more minutes to complete.
+====
+
+When you apply the `NodeObservability` CR, it creates the necessary machine config and machine config pool CRs to enable the CRI-O profiling on the worker nodes.
+
+[NOTE]
+====
+Kubelet profiling is enabled by default.
+====
+
+The CRIO unix socket of the node is mounted on the agent pod, which allows the agent to communicate with CRIO to run the pprof request. Similiarly, the `kubelet-serving-ca` certificate chain is mounted on the agent pod, which allows secure communication between the agent and node's kubelet endpoint.
```

Comment:
Should instances of CRIO in this paragraph be CRI-O?

Reply:
fixed.